_posts/2016/2016-06-14-gradle-jenkins-plugin.md

```diff
@@ -1,19 +1,20 @@
 ---
 layout: post
 title: "How to empower your Jenkins with Gradle"
-date: "2016-05-31 10:59:11"
+date: "2016-06-14 10:59:11"
+image: blog-head/gradle-jenkins-tools.jpeg
 categories: tech-stories
-authors: ["Renato T."]
+authors: ["Renato"]
 ---
 
-Since the [last blog]( https://developer.epages.com/blog/2016/01/28/jenkins-job-dsl-plugin.html) post we did some updates in our Jenkins environments and today we are going to talk a bit about it.
-We are using a [Gradle Jenkins plugin](https://github.com/ghale/gradle-jenkins-plugin) to control Jenkins installation.
+Since the [last blog post]( https://developer.epages.com/blog/2016/01/28/jenkins-job-dsl-plugin.html) post we did some updates in our Jenkins environments and today I'm going to talk a bit about it.
+We are using a [Gradle Jenkins plugin](https://github.com/ghale/gradle-jenkins-plugin) to control the Jenkins installation.
 
-We already mention about the dsl on the previous blogpost, you can find more information about the [Job DSL Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Job+DSL+Plugin)
+We already mention about the DSL on the previous blogpost, you can find more information about the [Job DSL Plugin](https://developer.epages.com/blog/2016/01/28/jenkins-job-dsl-plugin.html)
```

Comment:
We already mention about the DSL on the previous blogpost, you can find more information about the [Job DSL Plugin](https://developer.epages.com/blog/2016/01/28/jenkins-job-dsl-plugin.html)
>
hmmm, this is double, because it's already mentioned in the sentence above. Maybe delete it?